lib/url.js

```diff
@@ -80,10 +84,17 @@ const unsafeProtocol = new SafeSet([
   'javascript',
   'javascript:'
 ]);
+// Protocols that allows socketPath
+const socketPathProtocol = new SafeSet([
```

Comment:
```suggestion
const unixSocketOverHttpProtocol = new SafeSet([
```
May this be more proper, BTW do we have a https+unix?